docs/src/modules/java-protobuf/pages/actions-publishing-subscribing.adoc

```diff
@@ -636,3 +636,23 @@ include::example$scala-protobuf-eventsourced-counter/src/test/scala/com/example/
 <2> Clear the topic ignoring any unread messages.
 
 NOTE: Despite the example, you are neither forced to clear all topics nor to do it before each test. You can do it selectively, or you might not even need it depending on your tests and the flows they test.
+
+=== External Broker
+
+To run an integration test against a real instance of Google PubSub (or its Emulator), use the TestKit settings to override the default eventing support, as shown below:
+
+[.tabset]
+Java::
++
+[source,java]
+private static final KalixTestKitResource testKit = new KalixTestKitResource(
+        Main.createKalix(),
+        Settings.DEFAULT.withEventingSupport(EventingSupport.GOOGLE_PUBSUB)
+    );
+Scala::
++
+[source,scala,indent=0]
+private val testKit = KalixTestKit(
+    Main.createKalix(),
+    DefaultSettings.withEventingSupport(GooglePubSub)
+  ).start()
```

Comment:
Turns out that for protocol-first sdks the config is easier so I don't think we need to actually have samples for this. The snippets seem enough to me.